Android.mk

```diff
@@ -0,0 +1,23 @@
+LOCAL_PATH:= $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := Silence
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := Silence
+
+silence_root  := $(LOCAL_PATH)
+silence_build := $(silence_root)/build
+silence_apk   := build/outputs/apk/Silence-release-unsigned.apk
+
+$(silence_root)/$(silence_apk):
+	rm -Rf $(silence_build)
```

Comment:
Shouldn't that be in a `make clean` instruction?

Reply:
Android.mk is not a standard makefile, so there is no make clean there.